ReleaseNotes-2.10.7

```diff
@@ -0,0 +1,18 @@
+#
+# Open Build Service 2.10.7
+#
+
+Please read the README.md file for initial installation
+instructions or use the OBS Appliance from
+
+  http://openbuildservice.org/download/
+
+The dist/README.UPDATERS file has information for people updating
+from a previous OBS release.
+
+
+Bugfixes
+========
+
+* frontend:
```

Comment:
s/frontend/Frontend/

Previous release notes, 2.10.6, had it not capitalized. But the rest, yes.